yang/frr-bgp-filter.yang

```diff
@@ -0,0 +1,341 @@
+          }
+        }
+      }
+    }
+
+    list extcommunity-list {
+      key "name";
+      description
+        "Extcommunity-list instance";
+      leaf name {
+        type string;
+      }
+
+      list entry {
+        key "sequence";
+        description
+          "Extcommunity-list entry";
+        leaf sequence {
+          type list-sequence;
+        }
+
+        leaf action {
+          type list-action;
+        }
+
+        leaf type {
+          type enumeration {
+            enum "extcommunity-list-standard-id" {
+              value 0;
+              description
+                "Standard extcommunity-list identifier";
+            }
+            enum "extcommunity-list-extended-id" {
+              value 1;
+              description
+                "Expanded extcommunity-list identifier";
+            }
+            enum "extcommunity-list-standard-name" {
+              value 2;
+              description
+                "Standard extcommunity-list name";
+            }
+            enum "extcommunity-list-extended-name" {
+              value 3;
+              description
+                "Expanded extcommunity-list name";
+            }
+          }
+          mandatory true;
+          description
+            "Extcommunity-list instance name/identifier";
+        }
+
+        choice Extcommunity-string {
+          description
+            "Extcommunity string";
+          case standard {
+            when "./type = 'extcommunity-list-standard-id' or\n./type = 'extcommunity-list-standard-name'";
+            choice standard-extcommunity-string {
+              description
+                "Value of the ext-community";
+              case extcommunity-rt {
+                description
+                  "Set BGP ext-community route-target attribute";
+                leaf-list extcommunity-rt {
+                  type rt-types:route-target;
+                }
+              }
+
+              case extcommunity-soo {
+                description
+                  "Set BGP ext-community site-of-origin attribute";
+                leaf-list extcommunity-soo {
+                  type rt-types:route-target;
+                }
+              }
+            }
+          }
+
+          case expanded {
+            when "./type = 'extcommunity-list-extended-id' or\n./type = 'extcommunity-list-extended-name'";
+            leaf expanded-extcommunity-string {
+              type string;
+              description
+                "Extcommunity string reg-ex";
+            }
+          }
+        }
+      }
+    }
+
+    list as-path-list {
+      key "name";
+      description
+        "AS-path access-list instance";
+      leaf name {
+        type string;
+        description
+          "AS-path access-list instance name/identifier";
+      }
+
+      list entry {
+        key "sequence";
+        description
+          "AS-path access-list entry";
+        leaf sequence {
+          type list-sequence;
+        }
```

Comment:
There isn't a sequence number on the vtysh command. Is this a unique requirement of the northbound/yang methodology?